server/src/rooms/schema/Character.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  Schema,
+  Context,
+  type,
+  MapSchema,
+  ArraySchema,
+} from "@colyseus/schema";
+
+export abstract class Character extends Schema {
+  @type("number") x: number;
+  @type("number") y: number;
+  @type("string") pos: string;
+  @type("string") lastMovedTime: string;
```

Comment:
note how this lastMovedTime is string, but the curRoundStartTime in BattleRoomState is number. Should try to be consistent?